src/HelperTraits/GTINMigrationUtilities.php

```diff
@@ -84,4 +85,70 @@ protected function get_gtin_migration_status(): string {
 	protected function options() {
 		return $this->options ?? woogle_get_container()->get( OptionsInterface::class );
 	}
+
+	/**
+	 * Prepares the GTIN to be saved.
+	 *
+	 * @param string $gtin
+	 * @return string
+	 */
+	protected function prepare_gtin( string $gtin ) {
+		return str_replace( '-', '', $gtin );
+	}
+
+	/**
+	 * Gets the message when the GTIN is invalid.
+	 *
+	 * @param \WC_Product $product
+	 * @param string      $gtin
+	 * @return string
+	 */
+	protected function error_gtin_invalid( \WC_Product $product, string $gtin ) {
```

Comment:
Not a big deal, and we don't seem to be consistent. But in most cases in the code we add `use WC_Product` to be able to remove the `\`. But I'm not attached to either method so both are fine.